### PR TITLE
Re-did the songs and band names, as requested

### PR DESCRIPTION
**Pull request by Pointedstick**
Re-did the songs and band names, as requested. Songs and band names are no longer associated with one another like they were before, but there are now no hails mentioning both so there will never be repeats or TMBR songs associated with the wrong band. And separating them enables lots of hails that weren't possible before and allows for a vast increase in the number of bands and songs.

For some reason GitHub made the diff kind of weird but hopefully it should be clear enough what the changes are.

### CODE REVIEW

**Comment by Johny-Cash** [comment]
Might offend some people, but OK.

File: data/hails.txt [hunk 1]
```diff
@@ -8,244 +8,411 @@
...
+		"Catastrophe Aftermath"
+		"Swift of the Sponge"
+		"Shrapnel Vacuum"
+		"Daniel Suburbia"
+		"Volume Maneuver"
+		"Galactic Grouch"
+		"Greed"
+		"Hacker Headache"
+		"Diffusion Four"
+		"There Might Be Riots"
```
Comment (Pointedstick) [inline data/hails.txt]: Nope! Good catch.

File: data/hails.txt [hunk 2]
```diff
@@ -21,7 +21,6 @@ phrase "band"
 		"Greed"
 		"Hacker Headache"
 		"Diffusion Four"
-		"There Might Be Riots"
```
Comment (KiLEdEnNis) [inline data/hails.txt]: should have made it alphabetical :P

**Comment by Pointedstick** [comment]
Awesome, thanks!

**Pull request merged**